### DISCUSSION TITLE
Captions disabled but still auto-generate

### DISCUSSION THREAD

**Discussion by GitAlexToTheGreek** [libre-tube/LibreTube#6813] [Q&A] [answered]
Hi there 👋

Despite setting captions to "None", it's necessary to repeatedly set each newly played video to "None" over and over...

![Screenshot_20241122_153147_LibreTube](https://github.com/user-attachments/assets/f7c1ee6a-d54d-44de-a491-2bd53e61571a)

![Screenshot_20241122_154051_LibreTube](https://github.com/user-attachments/assets/dc1aa114-9a15-4a02-8729-11812cf4c6c9)

![Screenshot_20241122_153204_LibreTube](https://github.com/user-attachments/assets/f2263229-8f19-4f30-8e6a-9ddeb1756942)

![Screenshot_20241122_153557_LibreTube](https://github.com/user-attachments/assets/844a568e-358e-4235-b5da-fc196b54acb0)

👉 **Can anyone assist?**

**Comment by GitAlexToTheGreek** [comment]
🫤 Still happening with every video...

Can anyone advise?

**Comment by GitAlexToTheGreek** [comment]
Sorry for the bother, @Bnyro -

These auto-captions (despite disabling them) persist with every played video - is it possible to address this?

**Comment by Jeidnx** [comment]
Have you tried disabling the subtitles in the settings? They may be linked to your system settings, so it's worth checking them as well.

**Reply by GitAlexToTheGreek** [reply]
> Have you tried disabling the subtitles in the settings? They may be linked to your system settings, so it's worth checking them as well.

Hi @Jeidnx

Thanks for the feedback.

Please see this brief screen recording:

https://github.com/user-attachments/assets/8f303ea2-5a01-4b04-81ba-ecba9f2fdeb1

...to see what I mean; despite confirming in Settings that captions aren't enabled, then opening a video, seeing the visible captions, double-checking that captions are set to "None" (which, once clicked again as shown in the video, *does* effectively remove them for the duration of that currently playing vid), but then opening another video, the captions are visible again.

_**Edit:
If by "system settings" you're referring to the device's (Samsung S23 Ultra (SM-S918U) - Android 14) settings, I checked all relevant Captions sections there as well and nothing's enabled there either...plus other video apps like YouTube, Netflix, Prime Video, etc, do not exhibit this same behavior.**_

It's a fairly exasperating thing - hoping the devs can address this.

**Reply by Jeidnx** [reply, marked as answer]
The settings shown in the video are not for enabling captions - they are for syncing the captions design with the systems. Check the option "Subtitle language" a bit further down in the Defaults section, it should be set to None to disable captions.

**Reply by GitAlexToTheGreek** [reply]
> The settings shown in the video are not for enabling captions - they are for syncing the captions design with the systems. Check the option "Subtitle language" a bit further down in the Defaults section, it should be set to None to disable captions.

That did it! 👌👏👍

Much obliged.
